spec/factories/users.rb

```diff
@@ -5,6 +5,8 @@
     email { "test@test.test" }
     password { "validpassword" }
     roles { ROLE::USER_MANAGER }
+    cognito_groups { ['test'] }
+    team { 'test' }
```

Comment:
should it be an integer (for now) to map more accurrately how it's being used in the controller? i.e. `@team = Team.find_by_id(current_user.team)` (we should have called it `team_id` to avoid this inconsistency)

Reply:
changed to how the others do it which is to put a 1 in there.